angular/src/lib/popover/tests/__snapshots__/popover.component.spec.ts.snap

```diff
@@ -7,33 +7,41 @@ exports[`Popover Test should match snapshot 1`] = `
   <div
     class="cdk-overlay-connected-position-bounding-box"
     dir="ltr"
-    style="top: 0px; left: 0px; height: 0px; width: 0px; align-items: flex-start; justify-content: center;"
+    style="left: 0px; bottom: 0px; height: 0px; width: 0px; align-items: center; justify-content: flex-end;"
```

Comment:
I suggest to reduce the inline style in HTML template as many as possible. Some scrum team will do some security check to remove all the inline style. Using css class will be a better choice. And you can still use Javascript to change the dom style dynamically.

Reply:
this again is the cdk ..